Update README.md (#127)
yiisoft/config: README.md

[![type-coverage](https://shepherd.dev/github/yiisoft/config/coverage.svg)](https://shepherd.dev/github/yiisoft/config)

This [Composer](https://getcomposer.org/) plugin provides assembling of configurations distributed with composer

> packages. It is implementing a plugin system which allows to provide the configuration needed to use a package direclty when installing it to make it run out-of-the-box.

Reviewer: ```suggestion
packages. It is implementing a plugin system which allows to provide the configuration needed to use a package directly when installing it to make it run out-of-the-box.
```

}
```

> In the above example the mapping keys are config group names and the values are configuration files.

Reviewer 2: ```suggestion
In the above example the mapping keys are config group names and the values are configuration files and references to another configs.
```

> In the above example the mapping keys are config group names and the values are configuration files and references to another configs.

Author: ```suggestion
In the above example the mapping keys are config group names and the values are configuration files and references to other config groups.
```

> The file paths are relative to the [source-directory](#source-directory), which by default is the path where composer.json is located.

Reviewer 3: ```suggestion
The file paths are relative to the [source-directory](#source-directory), which by default is the path where `composer.json` is located.
```
